plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-shipping-method-block/block.tsx

```diff
@@ -128,8 +132,10 @@ const ShippingSelector = ( {
 		);
 
 	return (
-		<Radio
-			value="shipping"
```

Comment:
@samueljseay I noticed that for the previous radio you keep the `value="pickup"` whereas here you remove the `			value="shipping"`

Reply:
@ralucaStan thank you for spotting this. The value isn't needed for either because it's handled in creating the on click handler. i'll remove this for consistency

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/45974/commits/225809380d185b093bea7137e1654b13ac793526